test/database_test.dart

```diff
@@ -947,25 +947,25 @@ Future testSimpleQuery() async {
   expect(result.length, 4);
   expect(result[0]['my_field'], 6);
 
-  result = await collection.findOne(where.eq('my_field', 3));
-  expect(result, isNotNull);
-  expect(result['my_field'], 3);
-  id = result['_id'];
+  var result2 = await collection.findOne(where.eq('my_field', 3));
```

Comment:
if you're creating a new variable for this, then maybe all these `var`s should be `final` after all.